CMakeLists.txt

```diff
@@ -97,7 +97,10 @@ if(NOT CMAKE_CROSSCOMPILING)
         if(MSVC_VERSION GREATER 1699)
             add_definitions(/D_ALLOW_KEYWORD_MACROS)
         endif()
-        add_definitions(/Dinline=__inline)
+        if(NOT USE_CLANG_CL)
+            # FIXME: Inspect
+            add_definitions(/Dinline=__inline)
```

Comment:
i'm curious what this is for?
clang has the __inline keyword, or is it defined to be a no-op somewhere?

Reply:
I was not able to compile the host tools with this, I got errors with the MS public headers. You may disable this and enable the host tools compiler lines (that I have commented out) to see them.